0pdd.rb

```diff
@@ -352,6 +352,13 @@
   )
 end
 
+error Octokit::NotFound do
+  status 404
+  haml :repo_unavailable, layout: :layout, locals: merged(
+    title: 'This repository is unavailable'
```

Comment:
@tedtoer would be great to say which repository is not found. The name of the repo should be here.